spec/migrations/20170425154145_add_sti_to_middlewares_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_migration
+
+describe AddStiToMiddlewares do
+  constants = %w(
+    MiddlewareDatasource
+    MiddlewareDeployment
+    MiddlewareDomain
+    MiddlewareMessaging
+    MiddlewareServer
+    MiddlewareServerGroup
+  )
+
+  constants.each do |klass|
+    migration_context :up do
```

Comment:
Because of the way migration_context works, This will end up with multiple `before(:all)` and `it("with empty tables")` definitions, which may be problematic (or at a minimum, redundant).

Can you put the migration_context *outside* of the `constants.each`?